Source/Core/Core/AchievementManager.h

```diff
@@ -36,6 +41,15 @@ class AchievementManager
   bool IsLoggedIn() const;
   void LoadGameByFilenameAsync(const std::string& iso_path, const ResponseCallback& callback);
 
+  void LoadUnlockData(const ResponseCallback& callback);
+  void ActivateDeactivateAchievements();
+  void ActivateDeactivateLeaderboards();
+  void ActivateDeactivateRichPresence();
+
+  void DoFrame();
+  unsigned MemoryPeeker(unsigned address, unsigned num_bytes, void* ud);
```

Comment:
```suggestion
  u32 MemoryPeeker(u32 address, u32 num_bytes, void* ud);
```

u32 will always be equivalent to `unsigned int` and `unsigned` and is much shorter to read and write